backend/src/main/java/com/example/vehiclesservice/config/ModelMapperConfig.java

```diff
@@ -0,0 +1,16 @@
+package com.example.vehiclesservice.config;
+
+import org.modelmapper.ModelMapper;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Configuration
+public class ModelMapperConfig {
+    @Bean
+    public ModelMapper modelMapper() {
```

Comment:
You can autowire this bean in `GarageServiceImpl`, `RentingServiceImpl`, `VehicleServiceImpl` without creating a new mapper in each of these services.